### ISSUE TITLE
library.properties links do not work if the protocol is https

### ISSUE THREAD

**Issue by ryantm** [arduino/Arduino#2855]
Hi.

I am using Windows 8.1 Pro and Arduino 1.6.2. When I click on a "More info" link in the Library Manager it opens if the link has the HTTP protocol but not if it has HTTPS. For example, clicking on the Adafruit NeoPixel link https://github.com/adafruit/Adafruit_NeoPixel does not do anything, but clicking on the AltSoftSerial link http://www.pjrc.com/teensy/td_libs_AltSoftSerial.html opens the link in Google Chrome. I also verified the same behavior by changing the protocol of the URL in the library.properties file for my own library installed on my machine.

Ryan

**Comment by ffissore** [comment]
On my linux box, where firefox is the default, it opens adafruit link properly. I'll do some more tests with other setups

**Comment by ffissore** [comment]
One thing comes to mind: the link is not clickable on unselected row. You must first click on a row to select it and only then the link becomes clickable: is this the source of your issue?

**Comment by ryantm** [comment]
@ffissore Yes, I noticed that and I selected the row before I tried.

**Comment by ryantm** [comment]
I reproduced this problem on another co-worker's computer that is also Windows 8.1 Pro.

**Comment by ryantm** [comment]
I confirmed this is not a problem on Ubuntu 14.04
```
Linux 3.13.0-46-generic #75-Ubuntu SMP Tue Feb 10 15:24:04 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
````

**Comment by ffissore** [comment]
Found and fixed with https://github.com/arduino/Arduino/commit/3a916633b1e3ac2f54e6886b9693937aa9fcd442 will be available with next nightly

**Comment by ryantm** [comment]
Confirmed that 1.6.3 fixes this on Windows 8.1 Pro.
